lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts

```diff
@@ -117,4 +117,16 @@ export class LegalHoldService {
     bulkHold(holdId: string, query: string, language: string): Observable<BulkHoldAddResponse> {
```

Comment:
Can we keep the naming consistent and use `bulkAssignHold` or `bulkAssignHolds`?

Reply:
renamed to `bulkAssignHold`